src/modules/auth/clients/FormFields.js

```diff
@@ -18,11 +25,13 @@ export function TextInput ({ name, register, labelName, disabled }) {
 				name={name}
 				type="text"
 				disabled={disabled}
+				required={name == "client_name" ? true : false}
 				onChange={reg.onChange}
 				onBlur={reg.onBlur}
 				innerRef={reg.ref}
-
+				invalid={(errors?.preferred_client_id) && errors.preferred_client_id}
 			/>
+			{(errors?.preferred_client_id) && <FormFeedback>{errors.preferred_client_id?.message}</FormFeedback>}
```

Comment:
@Aringoaway `{errors.preferred_client_id && <FormFeedback>{errors.preferred_client_id?.message}</FormFeedback>}

should be enough, but please test that

Reply:
@Pe5h4 I tested it and `?` is aslo here is necessary `{errors?.preferred_client_id && <FormFeedback>{errors.preferred_client_id.message}</FormFeedback>}`

This is a check that should be here. Because I use this component for different inputs and only this input `preferred_client_id` has an error message.